src/main/java/ua/edu/internship/interview/config/migrations/SkillsInitializerChangeUnit.java

```diff
@@ -0,0 +1,66 @@
+package ua.edu.internship.interview.config.migrations;
+
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackExecution;
+import lombok.RequiredArgsConstructor;
+import org.bson.types.ObjectId;
+import ua.edu.internship.interview.data.documents.SkillDocument;
+import ua.edu.internship.interview.data.repository.SkillRepository;
+
+import java.util.List;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'
<!-- __reviewdog__:ChBiNDFjZjA3NDZhZGI5MGQ3EgpjaGVja3N0eWxl -->